varcode/variant_collection.py

```diff
@@ -110,49 +67,55 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
-    def clone(self, new_variants=None):
+    def clone_metadata(self, new_variants):
```

Comment:
Funny, I just changed it from `clone` to `clone_metadata` since Tavi felt like it's strange to allow the possibility that `x != x.clone(some_args)`.